examples/bsc-example/src/app/bsc.offchain.example.ts

```diff
@@ -1,34 +1,40 @@
 import { TatumBscSDK } from '@tatumio/bsc'
-import { REPLACE_ME_WITH_TATUM_API_KEY } from '@tatumio/shared-testing-common'
-
-const bscSdk = TatumBscSDK({ apiKey: REPLACE_ME_WITH_TATUM_API_KEY })
 
+const bscSdk = TatumBscSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
 export async function bscOffchainExample() {
-  const account = await bscSdk.offchain.depositAddress.checkExists(
-    '0xa7673161CbfE0116A4De9E341f8465940c2211d4',
-  )
-  const address = await bscSdk.offchain.depositAddress.create('0xa7673161CbfE0116A4De9E341f8465940c2211d4', 1)
-  const adresses = await bscSdk.offchain.depositAddress.createMultiple({
-    addresses: [
-      {
-        accountId: '5e6be8e9e6aa436299950c41',
-        derivationKey: 0,
-      },
-      {
-        accountId: '5e6be8e9e6aa436299951n35',
-        derivationKey: 1,
-      },
-    ],
+  // if you don't already have a wallet, address and private key - generate them
+  // https://apidoc.tatum.io/tag/BNB-Smart-Chain#operation/BscGenerateWallet
+  const { mnemonic, xpub } = await bscSdk.wallet.generateWallet()
+  // https://apidoc.tatum.io/tag/BNB-Smart-Chain#operation/BscGenerateAddressPrivateKey
+  const fromPrivateKey = await bscSdk.wallet.generatePrivateKeyFromMnemonic(mnemonic, 0)
```

Comment:
use private key with index 1, if you generate address with index 1. You are using address 1 and index 0